### PR TITLE
YAML - base

### PR DESCRIPTION
**Pull request by dmaziec**

### CODE REVIEW

File: portal_objects/software.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+commit: "954e607"
+source_url: https://github.com/vcftools/vcftools
+
+---
+
+################################################################
+# BCFtools, 1.11
+################################################################
+name: BCFtools
+commit: "1.11"
```
Comment (dmaziec) [inline portal_objects/software.yaml]:
```suggestion
version: "1.11"
```

File: dockerfiles/granite/scripts/sanitize_vcf.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Script to clean input VCF file")
+
+    parser.add_argument("-i", "--inputfile", help="input VCF file", required=True)
+    parser.add_argument("-o", "--outputfile", help="output VCF file", required=True)
+
+    args = vars(parser.parse_args())
+
+    main(args)
```
Comment (dmaziec) [inline dockerfiles/granite/scripts/sanitize_vcf.py:132]: can we add some tests for this script?

File: dockerfiles/granite/scripts/sanitize_vcf.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+##INFO=<ID=SVINSSEQ,Number=.,Type=String,Description="Sequence of insertion">
+##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
+##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
+##INFO=<ID=SwappedAlleles,Number=0,Type=Flag,Description="The REF and the ALT alleles have been swapped in liftover due to changes in the reference. It is possible that not all INFO annotations reflect this swap, and in the genotypes, only the GT, PL, and AD fields have been modified. You should check the TAGS_TO_REVERSE parameter that was used during the LiftOver to be sure.">
+
+################################################
+#   FUNCTIONS
+################################################
+def filter_header(definitions):
+    """
```
Comment (dmaziec) [inline dockerfiles/granite/scripts/sanitize_vcf.py:77]: missing description

File: dockerfiles/granite/scripts/sanitize_vcf.py (hunk 2 above)
Comment (B3rse) [reply dockerfiles/granite/scripts/sanitize_vcf.py:132]: We can do that

**Pull request merged**